css/index.css

```diff
@@ -54,6 +54,41 @@ table {
 html, body {
     height: 100%;
     font-family: 'Roboto', sans-serif;
+    width: 800px;
+    margin: 0 auto;
+}
+
+header {
+    display: flex;
+    margin: 20px 0;
+    align-items: flex-end;
+    justify-content: space-between;
+}
+
+header nav {
+    width: 620px;
+}
+
+header nav a {
+    text-decoration: none;
+    color: black;
+    padding: 20px 35px 8px 35px;
+    border-radius: 3px;
+}
+
+@keyframes color-fade {
+    from {background-color: white;}
+    to {background-color: black;}
```

Comment:
Great job learning the more difficult of the animations as well. The `@keyframes` syntax is significantly more powerful, and thus more difficult of the animation methods which is awesome to see you getting into!

Especially using the separation of concerns to make it so that animation can be applied to multiple selectors via the `animation-name` method rather than applied directly to each selector and rewritten each time.